docs/BestPractices.md

```diff
@@ -153,3 +154,15 @@ $ docker run \
 ## Security
 
 The Docker team has provided a tool to analyze your running containers for potential security issues. You can download and run this tool from here: https://github.com/docker/docker-bench-security
+
+## node-gyp alpine
+
+Here is an example of how you would install dependencies that need node-gyp support on alpine release:
+
+```Dockerfile
+FROM node:alpine
+
+RUN apk add --no-cache --virtual .gyp python make g++ \
```

Comment:
Maybe git as well?

Reply:
the tools installed are only the node-gyp dependencies, as described in the node-gyp docs:
https://github.com/nodejs/node-gyp#on-unix